net/redeemer.proto

```diff
@@ -6,9 +6,14 @@ package net;
 
 service TicketRedeemer {
     rpc QueueTicket(Ticket) returns (google.protobuf.Empty) {}
+    rpc MaxFloat(MaxFloatRequest) returns (MaxFloatUpdate) {}
```

Comment:
Curious, when is this `MaxFloat` RPC call invoked? If it's done alongside the `MonitorMaxFloat` subscriptions to initialize state, then can we just send down the max float immediately after subscribing, and avoid the separate RPC call?

Reply:
`MonitorMaxFloat` opens up a single stream to the client to send max float updates for all senders. This stream is started when the client initialises. 

`MaxFloat` would be called during e.g. initial discovery when there is no `maxFloat` value initialised yet for the sender on the client because it has not yet redeemed a winning ticket at the redeemer. This method would never be called during the transcoding workflow as the client should always have local cache available. 

I think what you describe could be possible now that I think of it if `client.MaxFloat()` opens up the subscription rather than when queueing the first ticket. We would have to create a `MonitorMaxFloatReq` message that contains the sender field. Will look further into it!

Reply:
Ah, I was thinking the workflow was something like this:

1. Orchestrator starts out monitoring nothing
1. When orchestrator receives a new job, invoke a rpc call to monitor for the max float for a specific broadcaster. (Would also immediately receive the current MaxFloat upon subscribing)
1. Receive new float values from redeemer service as necessary for that particular broadcaster
1. When job expires on orchestrator, halt subscription

Reply:
That does not work because discovery and receiving jobs are two different things. We also need `maxFloat` during discovery.

Reply:
Yep, gotcha - it makes sense why we'd need `MaxFloat` during the initial phase, eg discovery. In this case, the additional RPC call for a per-broadcaster subscription is less of a big deal - it can be initiated asynchronously when a job is received, and the cached value used in the meantime. But as mentioned in https://github.com/livepeer/go-livepeer/pull/1513#discussion_r432720399 , keeping RPC monitoring stateless is OK for now.